<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoBooth</class>
 <widget class="QWidget" name="PhotoBooth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>230</red>
        <green>187</green>
        <blue>171</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>229</red>
        <green>187</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>226</red>
        <green>184</green>
        <blue>167</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Amatic</family>
    <pointsize>20</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PhotoBooth</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>1645</x>
     <y>25</y>
     <width>250</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>ressources/logo_blanc_sur_transparent.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="compteur">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>1020</y>
     <width>300</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>654 photos restantes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="loading">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Forfty Scribble</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 180, 163, 255), stop:1 rgba(242, 199, 183, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widgetPrint" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <widget class="QLabel" name="nbPrintLabel">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>300</y>
      <width>180</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonIncrease">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>320</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Amatic</family>
      <pointsize>60</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="shortcut">
     <string>Alt+Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPrinter">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>450</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/boutonPrinter.png</normaloff>ressources/boutonPrinter.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>180</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRestart">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>640</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/boutonRestart.png</normaloff>ressources/boutonRestart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>180</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCancel">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>830</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/boutonCancel.png</normaloff>ressources/boutonCancel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>180</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDecrease">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>320</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Amatic</family>
      <pointsize>60</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="shortcut">
     <string>Alt+Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="viewer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1620</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetDevelopper" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>970</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="shortcut">
     <string>Alt+Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No warning/error</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetPhoto" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <widget class="QPushButton" name="buttonPhoto">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>460</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/boutonPhoto.png</normaloff>ressources/boutonPhoto.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>180</width>
      <height>180</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="countdown">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-100</y>
      <width>1620</width>
      <height>700</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>500</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="camView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1620</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lookUp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1620</width>
      <height>280</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>160</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Regardez ici ☝</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>buttonPhoto</zorder>
   <zorder>camView</zorder>
   <zorder>countdown</zorder>
   <zorder>lookUp</zorder>
  </widget>
  <widget class="QPushButton" name="veilleButton">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-20</y>
     <width>1960</width>
     <height>1120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Amatic</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(222, 180, 163, 255), stop:1 rgba(242, 199, 183, 255));
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Toucher l'écran pour démarrer</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>loading</zorder>
  <zorder>logo</zorder>
  <zorder>compteur</zorder>
  <zorder>widgetPhoto</zorder>
  <zorder>widgetDevelopper</zorder>
  <zorder>widgetPrint</zorder>
  <zorder>veilleButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
